JSDemos/Demos/Form/Validation/description.md

```diff
@@ -1,10 +1,34 @@
-﻿This demo shows how to&nbsp;validate the [Form][0] editors. [Validation rules][1] are specified for each form item using the **validationRules** property. All editors on&nbsp;the form are combined into a&nbsp;validation group defined by&nbsp;the form&rsquo;s [validationGroup][2] property. 
+﻿This demo shows how to validate Form editors. To apply validation rules to an editor, declare them in the [validationRules[]](/Documentation/ApiReference/UI_Components/dxValidator/Validation_Rules/) array. Specify [type](/Documentation/ApiReference/UI_Components/dxValidator/Validation_Rules/RequiredRule/#type) and other properties for each rule. 
 
 
-Note that the _&laquo;Register&raquo;_ button here does not implement a&nbsp;usual [onClick][3] event handler. Instead, it&nbsp;has the [useSubmitBehavior][4] property set to _true_. This setting tells the button to&nbsp;validate and submit the HTML form in&nbsp;which it&nbsp;is&nbsp;nested, with no&nbsp;further configuration required.
+The following validation rules are shown in this demo:  
+- [RequiredRule](/Documentation/ApiReference/UI_Components/dxValidator/Validation_Rules/RequiredRule/)  
+Demands that a validated editor has a value.
```

Comment:
Why do you use `Demands` here? I'm not sure how something, other than a person, can 'demand' something.

Reply:
Does the word 'Requires' fit better?

Reply:
Yes